### PR TITLE
Translate `c-api/{abstract,bool,coro,descriptor,utilities}.po`

### LINKED ISSUE #168: Translate `c-api/{abstract,bool,coro,descriptor,utilities}.po`

### PR DESCRIPTION
**Pull request by mattwang44**
fix #168

### CODE REVIEW

**Comment by mattwang44** [comment]
@allen91wu, would you be interested in reviewing this PR?

**Comment by allen91wu** [comment]
Okay, I would review this PR~

**Comment by mattwang44** [comment]
@allen91wu I just sent you an [invitation](https://github.com/python/python-docs-zh-tw/invitations) for adding you to the team. I will assign you as the reviewer after you press the acceptance button 👍🏽.

File: faq/installed.po [hunk 1]
```diff
@@ -75,51 +78,52 @@ msgid ""
...
 
 #: ../../faq/installed.rst:32
 msgid ""
 "Many Unix-compatible operating systems, such as macOS and some Linux "
 "distributions, have Python installed by default; it's included in the base "
 "installation."
 msgstr ""
-"許多相容於 Unix 系統，例如：maxOS 和 一些 Linux 發行版本預設安裝 Python；安"
-"裝時被包含在基本安裝功能內。"
+"許多相容於 Unix 系統，例如 maxOS 和一些 Linux 發行版本預設安裝 Python；它被包"
```
Comment (allen91wu) [inline faq/installed.po]: - typo: `maxOS` -> `macOS`
```suggestion
"許多相容於 Unix 系統，例如 macOS 和一些 Linux 發行版本預設安裝 Python；它被包"
```

File: c-api/utilities.po [hunk 2]
```diff
@@ -30,3 +33,6 @@ msgid ""
 "C, and parsing function arguments and constructing Python values from C "
 "values."
 msgstr ""
+"本章中的函式可用來執行各種工具任務，包括幫助 C 程式碼提升跨平臺可移植性、在 "
```
Comment (allen91wu) [inline c-api/utilities.po]: - `portable` 應被譯為 `可攜性`
 - Ref: https://terms.naer.edu.tw/detail/1284282/?index=3
```suggestion
"本章中的函式可用來執行各種工具任務，包括幫助 C 程式碼提升跨平臺可攜性 (portable)、在 "
```

File: c-api/descriptor.po [hunk 3]
```diff
@@ -4,38 +4,46 @@
...
 
 #: ../../c-api/descriptor.rst:35
 msgid ""
 "Return non-zero if the descriptor objects *descr* describes a data "
 "attribute, or ``0`` if it describes a method.  *descr* must be a descriptor "
 "object; there is no error checking."
 msgstr ""
+"如果 descriptor 物件 *descr* 描述的是一個資料屬性則返回非零值，或者如果它描述"
+"的是一個方法則返回 ``0``\\ 。\\ *descr* 必須為一個 descriptor 物件；沒有錯誤"
+"檢測。"
```
Comment (allen91wu) [inline c-api/descriptor.po]:
```suggestion
"檢查。"
```

File: faq/installed.po [hunk 4]
```diff
@@ -75,51 +78,52 @@ msgid ""
...
 #: ../../faq/installed.rst:45
 msgid ""
 "If Python was installed by a third-party application, you can also remove "
 "it, but that application will no longer work.  You should use that "
 "application's uninstaller rather than removing Python directly."
 msgstr ""
-"若Python是透過第三方元件應用程式被安裝時，也可自行移除，不過請小心該應用程式"
-"將無法正常執行。你應該使用應用程式反安裝功能而非自行刪除Python目錄"
+"若 Python 是透過第三方應用程式安裝時，你也可自行移除，不過該應用程式將無法正"
+"常執行。你應該使用應用程式解除安裝功能而非自行刪除 Python。"
```
Comment (allen91wu) [inline faq/installed.po]:
```suggestion
"常執行。你應該使用應用程式解除安裝功能而非直接刪除 Python。"
```

File: faq/installed.po [hunk 5]
```diff
@@ -75,51 +78,52 @@ msgid ""
...
 #: ../../faq/installed.rst:29
 msgid ""
 "Some Windows machines also have Python installed.  At this writing we're "
 "aware of computers from Hewlett-Packard and Compaq that include Python.  "
 "Apparently some of HP/Compaq's administrative tools are written in Python."
 msgstr ""
-"一些安裝Windows的機器也被安裝Python。截至目前此文件交付，我們得知HP 出廠的機"
-"器預設安裝Python。顯然的HP管理工具程式是透過Python語言所撰寫。"
+"一些安裝 Windows 的機器也被安裝 Python。截至撰寫此文件的當下，我們得知 HP 與 "
+"Compaq 出廠的機器都預設安裝 Python，顯然的 HP 與 Compaq 部份管理工具程式是透"
```
Comment (allen91wu) [inline faq/installed.po]:
```suggestion
"Compaq 出廠的機器都預設安裝 Python。顯然的 HP 與 Compaq 部分的管理工具程式是透"
```

File: faq/installed.po [hunk 6]
```diff
@@ -128,6 +132,6 @@ msgid ""
 "and some of them might be important to you.  Reinstalling the whole system "
 "would then be required to fix things again."
 msgstr ""
-"當作業系統預設安裝Python，不建議移除它。對你而言某些工具程式是重要不可或缺"
-"的，若自行移除它，透過Python撰寫的工具程式將無法正常執行。重新安裝整個系統仍"
-"需再次處理這些事情。"
+"當作業系統預設安裝 Python，不建議移除它。對你而言某些工具程式是重要不可或缺"
+"的，若自行移除它，透過 Python 撰寫的工具程式將無法正常執行。重新安裝整個系統"
+"仍需再次處理這些事情。"
```
Comment (allen91wu) [inline faq/installed.po]:
```suggestion
"，才能再次解決這些問題。"
```

File: c-api/descriptor.po [hunk 7]
```diff
@@ -4,38 +4,46 @@
...
-msgstr ""
+msgstr "內建 descriptor 型別的型別物件。"
 
 #: ../../c-api/descriptor.rst:35
 msgid ""
 "Return non-zero if the descriptor objects *descr* describes a data "
 "attribute, or ``0`` if it describes a method.  *descr* must be a descriptor "
 "object; there is no error checking."
 msgstr ""
+"如果 descriptor 物件 *descr* 描述的是一個資料屬性則返回非零值，或者如果它描述"
```
Comment (allen91wu) [inline c-api/descriptor.po]:
```suggestion
"如果 descriptor 物件 *descr* 描述的是一個資料屬性則回傳非零值，或者如果它描述"
```

File: c-api/bool.po [hunk 8]
```diff
@@ -30,39 +33,49 @@ msgid ""
 "normal creation and deletion functions don't apply to booleans.  The "
 "following macros are available, however."
 msgstr ""
+"Python 中的 boolean 是以整數子類別化來實現的。只有 :const:`Py_False` 和 :"
+"const:`Py_True` 兩個 boolean，因此一般的建立和刪除函式並不適用於 boolean，但"
```
Comment (allen91wu) [inline c-api/bool.po]:
```suggestion
"const:`Py_True` 兩個 boolean。因此一般的建立和刪除函式並不適用於 boolean。但"
```

File: c-api/bool.po [hunk 9]
```diff
@@ -30,39 +33,49 @@ msgid ""
...
 "succeeds."
 msgstr ""
+"如果 *o* 的型別為 :c:data:`PyBool_Type` 則回傳真值。此函式總是會成功執行。"
 
 #: ../../c-api/bool.rst:22
 msgid ""
 "The Python ``False`` object.  This object has no methods.  It needs to be "
 "treated just like any other object with respect to reference counts."
 msgstr ""
+"Python 的 ``False`` 物件。此物件沒有任何方法，在參照 (reference) 計數上必須有"
```
Comment (allen91wu) [inline c-api/bool.po]:
```suggestion
"Python 的 ``False`` 物件。此物件沒有任何方法。在參照 (reference) 計數上必須有"
```

File: c-api/bool.po [hunk 10]
```diff
@@ -30,39 +33,49 @@ msgid ""
...
+"著和其他物件一樣的處理方式。"
 
 #: ../../c-api/bool.rst:28
 msgid ""
 "The Python ``True`` object.  This object has no methods.  It needs to be "
 "treated just like any other object with respect to reference counts."
 msgstr ""
+"Python 的 ``True`` 物件。此物件沒有任何方法，在參照計數上必須有著和其他物件一"
```
Comment (allen91wu) [inline c-api/bool.po]:
```suggestion
"Python 的 ``True`` 物件。此物件沒有任何方法。在參照計數上必須有著和其他物件一"
```

**Review by allen91wu — CHANGES_REQUESTED** [review]
@mattwang44 I have left some comments. If you have any questions please let me know. 🙏

**Review by allen91wu — APPROVED** [review]
Looks good!

**Pull request merged**